LoopKit/PumpManager.swift

```diff
@@ -14,16 +14,15 @@ public enum PumpManagerResult<T> {
     case failure(Error)
 }
 
+public protocol PumpManagerStatusObserver: class {
```

Comment:
They're identical now.  I did read that the proposal was to deprecate `class`, but it seems clearer to me to use class, since we're depending on memory management effects that apply to classes and not value types.

Reply:
Okay, works for me. (I think `AnyObject` can only refer to classes while `Any` can refer to classes or value types.)